src/DnsClient/DnsQueryOptions.cs

```diff
@@ -399,6 +399,12 @@ public LookupClientOptions(params IPAddress[] nameServers)
         /// </remarks>
         public bool AutoResolveNameServers { get; set; } = true;
 
+        /// <summary>
+        /// Gets or sets a flag indicating whether the Name Resolution Policy table in Windows should be evaluated
+        /// while trying to discover which nameservers should be used automatically.
+        /// </summary>
+        public bool UseNameResolutionPolicyTable { get; set; } = true;
```

Comment:
I'm not sure we really have to introduce another configuration point for this.
The current auto resolve mechanism does its best getting all kind of information already and most systems probably don't even have those policies, so I guess we could just always check for that?

Would also make my life easier with the constructor change of NameServer. 
That would be kind of a breaking change if we introduce another optional parameter -.-